docs/build-the-docs.md

```diff
@@ -366,8 +357,7 @@ Overwriting the default URL to local is especially helpful if you also want to c
 
 ### Searching and Fixing Attribute Errors
 
-It is very beneficial to use command line attributes when searching and fixing attribute errors. This can be
-necessary when you get warnings like: `WARNING: skipping reference to missing attribute: <attribute-name>`
+It is very beneficial to use command line attributes when searching and fixing attribute errors. This can be necessary when you get warnings like: `WARNING: skipping reference to missing attribute: <attribute-name>`
```

Comment:
```suggestion
It is very beneficial to use command-line attributes when searching and fixing attribute errors. This can be necessary when you get warnings like: `WARNING: skipping reference to missing attribute: <attribute-name>`.
```